com.jetbrains.PyCharm-Professional.yaml

```diff
@@ -0,0 +1,93 @@
+app-id: com.jetbrains.PyCharm-Professional
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: pycharm
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --socket=session-bus
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=xdg-run/keyring
+  - --device=all
+  - --env=PYCHARM_JDK=${FLATPAK_DEST}/pycharm/jre64/
+modules:
+  - shared-modules/glew/glew.json
+  - shared-modules/glu/glu-9.json
+
+  - name: freeglut
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DFREEGLUT_BUILD_STATIC_LIBS=OFF
+      - -DFREEGLUT_BUILD_DEMOS=OFF
+    sources:
+      - type: archive
+        url: "https://prdownloads.sourceforge.net/freeglut/freeglut-3.2.1.tar.gz"
+        sha256: d4000e02102acaf259998c870e25214739d1f16f67f99cb35e4f46841399da68
+
+  - name: python3-virtualenv
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} virtualenv
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/e7/80/15d28e5a075fb02366ce97558120bb987868dab3600233ec7be032dc6d01/virtualenv-16.7.7.tar.gz
+        sha256: d257bb3773e48cac60e475a19b608996c73f4d333b3ba2e4e57d5ac6134e0136
+      - type: file
+        url: https://files.pythonhosted.org/packages/f4/d5/a6c19dcbcbc267aca376558797f036d9bcdff344c9f785fe7d0fe9a5f2a7/setuptools-41.4.0.zip
+        sha256: 7eae782ccf36b790c21bde7d86a4f303a441cd77036b25c559a602cf5186ce4d
+      - type: file
+        url: https://files.pythonhosted.org/packages/59/b0/11710a598e1e148fb7cbf9220fd2a0b82c98e94efbdecb299cb25e7f0b39/wheel-0.33.6.tar.gz
+        sha256: 10c9da68765315ed98850f8e048347c3eb06dd81822dc2ab1d4fde9dc9702646
+
+  - name: python3-pipenv
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} pipenv
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/13/b4/3ffa55f77161cff9a5220f162670f7c5eb00df52e00939e203f601b0f579/pipenv-2018.11.26-py3-none-any.whl
+        sha256: 56ad5f5cb48f1e58878e14525a6e3129d4306049cb76d2f6a3e95df0d5fc6330
+      - type: file
+        url: https://files.pythonhosted.org/packages/ba/f8/50c2b7dbc99e05fce5e5b9d9a31f37c988c99acd4e8dedd720b7b8d4011d/virtualenv_clone-0.5.3-py2.py3-none-any.whl
+        sha256: 532f789a5c88adf339506e3ca03326f20ee82fd08ee5586b44dc859b5b4468c5
+      - type: file
+        url: https://files.pythonhosted.org/packages/18/b0/8146a4f8dd402f60744fa380bc73ca47303cccf8b9190fd16a827281eac2/certifi-2019.9.11-py2.py3-none-any.whl
+        sha256: fd7c7c74727ddcf00e9acd26bba8da604ffec95bf1c2144e67aff7a8b50e6cef
+
+  - name: pycharm
+    buildsystem: simple
+    build-commands:
+      - install -D apply_extra /app/bin/apply_extra
```

Comment:
Install the apply_extra after installing the desktop/metainfo/icons.

Reply:
Can do, but why?